src/main/java/emu/grasscutter/server/http/handlers/GenericHandler.java

```diff
@@ -41,7 +41,7 @@ public final class GenericHandler implements Router {
         express.all("/perf/config/verify", new HttpJsonResponse("{\"code\":0}"));
 
         // webstatic-sea.hoyoverse.com
-        express.get("/admin/mi18n/plat_oversea/*", new HttpJsonResponse("{\"version\":51}"));
+        express.get("/admin/mi18n/plat_oversea/m202003048/m202003048-version.json", new HttpJsonResponse("{\"version\":51}"));
```

Comment:
The reason this is a wildcard is because there are multiple versions of .json, typically it differs per platform but there are some other difference that I don't particularly know about.
What exactly is the issue with the current wildcard solution? (other then there are other files (-en.json) that need to be accessed, but I will be coming up with a fix for that soon.)

Reply:
#1005 i think

Reply:
> #1005 i think

Yeah I think that is related to the thing I mentioned about -en.json and related files.
When I made the wildcard fix before I didn't realise that there were translation files within there as well.
I already have a fix in mind for this without removing the wildcard and causing a bunch of other issues. Hopefully I will have it done later today.

Reply:
This is the type of file I didn't realise were needed
https://webstatic-sea.hoyoverse.com/admin/mi18n/plat_oversea/m202003048/m202003048-en.json